src/packages/tag/tag.taro.tsx

```
@@ -132,23 +132,24 @@ export const Tag: FunctionComponent<Partial<TagProps>> = (props) => {
-              >
-                X
-              </Text>
+              <>
+                <Close
+                  size={8}
+                  onClick={(e) => {
+                    setVisible(false)
+                    onClose && onClose(e)
+                  }}
+                />
+                <Text
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)